Src/VimCore/CommonOperations.fs

```diff
@@ -160,6 +160,18 @@ type internal CommonOperations
         with get() = _maintainCaretColumn
         and set value = _maintainCaretColumn <- value
 
+    /// Get the common operations for the specified text view
+    member x.GetCommonOperationsForTextView textView =
+        _vim.GetOrCreateVimBuffer textView

```

Comment:
Calling `GetOrCreateVimBuffer` directly can cause `IVimBuffer` to be created for `ITextView` that the host doesn't want them created for. Consider the powershell or immediate window as an example. The host has to make a decision as to whether or not the creation should take place. Should use `TryGetOrCreateVimBufferForHost` here instead. That will complicate this API a bit but think it's necessary.

Reply:
Ah, that makes sense.